bitnami/ghost/templates/deployment.yaml

```
@@ -11,9 +11,9 @@ spec:
   template:
     metadata:
       labels: {{- include "common.labels.standard" . | nindent 8 }}
-    {{- if .Values.podAnnotations }}
-    annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.podAnnotations "context" $ ) | nindent 6 }}
-    {{- end }}
+      {{- if .Values.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.podAnnotations "context" $ ) | nindent 6 }}
```

Moving the annotation to the correct place, the indentation should be also increased (same approach used for labels):
```suggestion
      annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.podAnnotations "context" $ ) | nindent 8 }}
```

---

whoops, good spot. thank you!